analyzere/base_resources.py

```diff
@@ -408,6 +408,13 @@ def back_allocation(self, source_id, auto_retry=True, **params):
         return convert_to_analyzere_object(data)
 
 
+class EventCatalogResource(DataResource):
+    def profile(self):
+        path = '%s/profile' % self._get_path(self.id)
```

Comment:
might as well keep the terminating slash on all hard-coded urls.  we do have the redirect, but I think it's better to remember that django wants you to have a slash by default.

Reply:
I'm going to leave it as it is to be consistent with the other sub resources.